string id
float32 profile_variance
float32 aoa_azimuth
float32 aoa_elevation
